import-export-cli/cmd/deletePolicy.go

```diff
@@ -0,0 +1,46 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update the license header with the year and Inc to LLC in here as well?